mu4/palette/internal/palette/paletteworkspace.cpp

```diff
@@ -902,11 +912,11 @@ bool PaletteWorkspace::resetPalette(const QModelIndex& index)
         =(MScore::noGui && MScore::testMode)
           ? QMessageBox::Yes
           : QMessageBox::question(
-              nullptr,
-              "",
-              tr("Do you want to restore this palette to its default state? All changes to this palette will be lost."),
-              QMessageBox::Yes | QMessageBox::No
-              );
+        nullptr,
+        "",
+        tr("Do you want to restore this palette to its default state? All changes to this palette will be lost."),
+        QMessageBox::Yes | QMessageBox::No
+        );
```

Comment:
Better change to use Interactive

Reply:
replaced all qt dialogs (in the palette module) with using of interactive